registry-scanner/pkg/env/env.go

```diff
@@ -35,6 +36,18 @@ func GetStringVal(envVar string, defaultValue string) string {
 	}
 }
 
+// GetDurationVal retrieves a time.Duration value from given environment envVar
+// Returns default value if envVar is not set.
+func GetDurationVal(envVar string, defaultValue time.Duration) time.Duration {
+	if val := os.Getenv(envVar); val != "" {
+		duration, err := time.ParseDuration(val)
+		if err == nil {
+			return duration
+		}
```

Comment:
maybe we should log a warning for invalid value?

Reply:
Yep, please see #1110